SwiftyJSON.xcodeproj/project.pbxproj

```diff
@@ -757,7 +757,7 @@
 			buildSettings = {
 				APPLICATION_EXTENSION_API_ONLY = YES;
 				BITCODE_GENERATION_MODE = bitcode;
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "";
```

Comment:
Can you help revert this changes?
I believe we wanna keep them as "iPhone Developer". 😄